### PR TITLE
fix: image alts

### PR DESCRIPTION
**Pull request by vinckr**
### Related issue

closes https://github.com/ory/web/issues/243

### CODE REVIEW

File: src/pages/kratos.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ const IndexPage = () => {
           <img
             className="responsive"
             loading="lazy"
-            alt=""
+            alt="Representation of Kratos UIs"
```
Comment (jfcurran) [inline src/pages/kratos.tsx:84]: maybe adding terms like `flexible` or `headless` might be smart here

File: src/pages/oathkeeper.tsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ const IndexPage = () => {
           <img
             className="responsive"
             loading="lazy"
-            alt=""
+            alt="Representation of Oathkeeper UIs"
```
Comment (jfcurran) [inline src/pages/oathkeeper.tsx:88]:
```suggestion
            alt="Example use of Oathkeeper IAM capabilities inside a UI"
```

File: src/pages/kratos.tsx (hunk 1 above)
Comment (jfcurran) [reply src/pages/kratos.tsx:84]:
```suggestion
            alt="Examples of headless Login and Registrations forms with different styles using Kratos"
```

File: src/pages/keto.tsx [hunk 3]
```diff
@@ -111,7 +111,12 @@ const IndexPage = () => {
           'Manage user roles, rights, and permissions with ACL based on Google Zanzibar.'
         }
         image={
-          <img className="responsive" loading="lazy" alt="" src={ketoProcess} />
+          <img
+            className="responsive"
+            loading="lazy"
+            alt="Representation of Keto UIs"
```
Comment (jfcurran) [inline src/pages/keto.tsx:117]:
```suggestion
            alt="Example use of Keto authorisation methods inside a UI"
```

File: src/pages/index.tsx [hunk 4]
```diff
@@ -197,7 +197,7 @@ const IndexPage = () => {
             height={'100%'}
             loading="lazy"
             className="responsive"
-            alt=""
+            alt="Example of integrating Ory"
```
Comment (jfcurran) [inline src/pages/index.tsx:200]:
```suggestion
            alt="Example code and terminal response when integrating Ory"
```

File: src/pages/hydra.tsx [hunk 5]
```diff
@@ -77,7 +77,7 @@ const IndexPage = () => {
           <img
             className="responsive"
             loading="lazy"
-            alt=""
+            alt="Representation of Hydra UIs"
```
Comment (jfcurran) [inline src/pages/hydra.tsx:80]:
```suggestion
            alt="Example OAuth2 flow with headless UI using Hydra"
```

**Review by jfcurran — COMMENTED** [review]
What do you think about these suggestions? I believe adding more specifics to the `alt` tags is beneficial when it comes to SEO, and it also makes it a lot more accessible :)

File: src/pages/kratos.tsx (hunk 1 above)
Comment (vinckr) [reply src/pages/kratos.tsx:84]: Right! I was not sure how long, elaborate the alt tag can be, so I opted for the shortest possible alt, that still explains the content.

**Review by vinckr — COMMENTED** [review]
LGTM

**Review by aeneasr — APPROVED** [review]

**Pull request merged**